src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -870,8 +870,12 @@ async function main(): Promise<void> {
 }
 
 if (require.main === module) {
-  main().catch((err) => {
-    new EmulatorLog("FATAL", "runtime-error", err.stack ? err.stack : err).log();
-    process.exit();
-  });
+  main()
```

Comment:
Not required, but, couldn't this be written:

```typescript
try {
  await main();
} catch (err) {
  new EmulatorLog...
  process.exit(1);
}
// I don't know if you'd _really_ need this, but probably wouldn't hurt:
process.exit(0);
```

Reply:
Nah because we can't have top-level `await` :-(